pkgs/stdenv/freebsd/default.nix

```diff
@@ -1,276 +1,460 @@
-      url = "https://ftp.gnu.org/gnu/${name}/${name}-${ver}.tar.gz";
-      sha256 = "08vz660768mnnax7n8d4d85jxafwdmsxsi7fh0hzvmafbvn9wkb0";
-      configureArgs = [ "--disable-nls"
-                        "--without-libintl-prefix"
-                        "--without-libiconv-prefix"
-                      ];
-    };
-    coreutils = trivialBuilder rec {
-      inherit (localSystem) system;
-      name = "coreutils";
-      ver = "8.31";
-      url = "https://ftp.gnu.org/gnu/${name}/${name}-${ver}.tar.xz";
-      sha256 = "1zg9m79x1i2nifj4kb0waf9x3i5h6ydkypkjnbsb9rnwis8rqypz";
-      configureArgs = [ "--disable-nls"
-                        "--without-libintl-prefix"
-                        "--without-libiconv-prefix"
-                        "--without-gmp"
-                        "--without-libpth-prefix"
-                      ];
-    };
-    findutils = trivialBuilder rec {
-      inherit (localSystem) system;
-      name = "findutils";
-      ver = "4.7.0";
-      url = "https://ftp.gnu.org/gnu/${name}/${name}-${ver}.tar.xz";
-      sha256 = "16kqz9yz98dasmj70jwf5py7jk558w96w0vgp3zf9xsqk3gzpzn5";
-      configureArgs = [ "--disable-nls"
-                        "--without-libintl-prefix"
-                        "--without-libiconv-prefix"
-                        "--without-gmp"
-                        "--without-libpth-prefix"
-                      ];
-    };
-    diffutils = trivialBuilder rec {
-      inherit (localSystem) system;
-      name = "diffutils";
-      ver = "3.7";
-      url = "https://ftp.gnu.org/gnu/${name}/${name}-${ver}.tar.xz";
-      sha256 = "09isrg0isjinv8c535nxsi1s86wfdfzml80dbw41dj9x3hiad9xk";
-      configureArgs = [ "--disable-nls"
-                        "--without-libintl-prefix"
-                        "--without-libiconv-prefix"
-                        "--without-libsigsegv-prefix"
-                      ];
-    };
-    grep = trivialBuilder rec {
-      inherit (localSystem) system;
-      name = "grep";
-      ver = "3.4";
-      url = "https://ftp.gnu.org/gnu/${name}/${name}-${ver}.tar.xz";
-      sha256 = "1yy33kiwrxrwj2nxa4fg15bvmwyghqbs8qwkdvy5phm784f7brjq";
-      configureArgs = [ "--disable-nls"
-                        "--without-libintl-prefix"
-                        "--without-libiconv-prefix"
-                        "--disable-perl-regexp"
-                        "--without-libsegsegv-prefix"
-                      ];
-    };
-    patch = trivialBuilder rec {
-      inherit (localSystem) system;
-      name = "patch";
-      ver = "2.7.6";
-      url = "https://ftp.gnu.org/gnu/${name}/${name}-${ver}.tar.xz";
-      sha256 = "1zfqy4rdcy279vwn2z1kbv19dcfw25d2aqy9nzvdkq5bjzd0nqdc";
+  all-bootstrap-urls-table = {
+    # These bootstrap hashes generated by compiling from nixpkgs 46c6e8a75f2329633858ca13a589698fcbf6231c on x86_64-linux
+    x86_64-freebsd = {
+      bash = {
+        url = "https://f005.backblazeb2.com/file/nixbsd/aldkfyqiq0nspljgl5pr0pgvxq99qirk-x86_64-freebsd-bash";
```

Comment:
We can even replace this with NixOS foundation hosted binaries before this PR is merged, I think. I will ask.

Reply:
Ok. So we need someone that can upload to tarballs.nixos.org i.e. @lovesegfault
Furthermore in the ideal case, we could bootstrap this tarball from an existing nixpkgs target for example Linux. Is this possible?

Reply:
as indicated by the comment, the tarball was bootstrapped from an existing nixpkgs target, x86_64-linux :)

The scripts for building the bootstrap tarball and generating this config are committed right now. I don't know if there's a kosher way to integrate them with hydra, but I'll gladly make it work.

Reply:
If we can cross-compile them, than we can manually upload the first version and for future versions we have scripts to regenerate bootstrap tarballs in nixpkgs.

Reply:
I contacted @lovesegfault again on matrix in the hopes, he would respond.